templates/server-ingress.yaml

```
@@ -8,8 +8,9 @@
 {{- $serviceName = printf "%s-%s" $serviceName "active" -}}
 {{- end }}
 {{- $servicePort := .Values.server.service.port -}}
-{{ if .Capabilities.APIVersions.Has "networking.k8s.io/v1beta1" }}
-apiVersion: networking.k8s.io/v1beta1
+{{- $kubeVersion := .Capabilities.KubeVersion.Version }}
+{{ if semverCompare ">= 1.19" $kubeVersion }}
+apiVersion: networking.k8s.io/v1
 {{ else }}
```

I think the semver version needs to be 1.19.0-0, to avoid problems with pre-release style version strings, as in GKE: https://github.com/helm/helm/issues/3810

Also, just so we don't change the API version unexpectedly on folks, can we do something like this?

```suggestion
{{- $kubeVersion := .Capabilities.KubeVersion.Version }}
{{ if semverCompare ">= 1.19.0-0" $kubeVersion }}
apiVersion: networking.k8s.io/v1
{{ else if .Capabilities.APIVersions.Has "networking.k8s.io/v1beta1" }}
apiVersion: networking.k8s.io/v1beta1
{{ else }}
```